keyboards/tada68/keymaps/abishalom/keymap.c

```diff
@@ -0,0 +1,98 @@
+#include "tada68.h"
```

Comment:
Please change this to:

```suggestion
#include QMK_KEYBOARD_H
```

That will include all the files you need automatically.